snc.sh

```diff
@@ -17,10 +17,10 @@ SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_rsa_c
 # If user defined the OPENSHIFT_VERSION environment variable then use it.
 # Otherwise use the tagged version if available
 function get_openshift_version {
-    if [ ${OPENSHIFT_VERSION} != "" ]; then
+    if [ "${OPENSHIFT_VERSION}X" != "X" ]; then
```

Comment:
@cardil why do you think `X` is required why not just compare `"${OPENSHIFT_VERSION}"` ?

Reply:
According to https://stackoverflow.com/questions/6852612/test-for-empty-string-with-x 
`[ "X${OPENSHIFT_VERSION}" != "X" ]` is better

Reply:
Shellcheck also raports this as invalid.

```bash
$ shellcheck snc.sh

In snc.sh line 19:
    if [ ${OPENSHIFT_VERSION} != "" ]; then
         ^-- SC2086: Double quote to prevent globbing and word splitting.
```

I propose this because without defining `OPENSHIFT_VERSION` script throws:

```
~/git/snc/snc.sh: 20: [: !=: unexpected operator
``` 

Shellcheck proposes to quote and I will do that. That also prevents an error without additional `X` sign, that might be mind bending.